lib/cdo/chat_client.rb

```diff
@@ -17,7 +17,7 @@ def self.developers(message, options={})
   end
 
   def self.log(message, options={})
-    message(CDO.hip_chat_log_room, message, options)
```

Comment:
It seems unnecessary for this to be specified by a `CDO` value, I see no reason why it shouldn't be hard-coded.